src/full_space.jl

```diff
@@ -196,7 +196,7 @@ end
 
 function NLPModels.jac_coord!(nlp::FullSpaceModel, x::AbstractVector, vals::AbstractVector)
     @lencheck nlp.meta.nvar x
-    @lencheck nlp.meta.nnzj vals
+    # @lencheck nlp.meta.nnzj vals
```

Comment:
Maybe add a small comment to remember we comment out this line because of MadNLP?